persistence/CloudProxyWrapper.js

```diff
@@ -9,13 +12,31 @@ class CloudProxyWrapper {
     connect() {
     }
 
+    localToRemote(path) {
+        if (path.startsWith(objectsPath)) {
+            let relPath = pathOps.relative(objectsPath, path);
+            if (!relPath.startsWith('..')) {
+                return relPath;
+            }
+        }
+        return path;
+    }
+
+    remoteToLocal(path) {
+        if (path.startsWith('/') || path.startsWith('..')) {
```

Comment:
Might want to also cover `~` at the start of the path

Reply:
Is anything going on here with pathOps.relative that I'm not seeing?

Reply:
Sort of, but I've adjusted the function to be clearer in what it's going for